packages/swirl-components/src/components/swirl-toolbar/swirl-toolbar.mdx

```diff
@@ -0,0 +1,32 @@
+import { Controls, Canvas, Meta, Story } from "@storybook/addon-docs";
+import * as Stories from "./swirl-toolbar.stories";
+
+<Meta title="Components/SwirlToolbar" />
+
+# SwirlToolbar
+
+The SwirlToolbar component is used to provide a set of actions that are relevant
+for the given context. It has to be used in conjunction with the
+[SwirlButton](?path=/docs/components-swirlbutton--docs) and / or
+[SwirlChip](?path=/docs/components-swirlchip--docs) component.
+
+- **[Figma Design Specs](#)**
+- **[Source Code](https://github.com/flip-corp/swirl/tree/main/packages/swirl-components/src/components/swirl-toolbar)**
+
+## Usage
+
+<Canvas of={Stories.SwirlToolbar} sourceState="shown"></Canvas>
+
+<Controls of={Stories.SwirlToolbar} />
+
+## Accessibility
+
```

Comment:
**suggestion:** Add a reference to the WAI page, like here: https://swirl-storybook.flip-app.dev/?path=/docs/components-swirldialog--docs

"The component follows the [WAI-ARIA Toolbar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/)."